sippy/UaStateConnected.py

```diff
@@ -56,8 +56,8 @@ def recvRequest(self, req):
                 self.ua.global_config['_sip_tm'].sendResponse(req.genResponse(400, 'Bad Request', server = self.ua.local_ua))
                 return None
             self.ua.global_config['_sip_tm'].sendResponse(req.genResponse(202, 'Accepted', server = self.ua.local_ua))
-            also = req.getHFBody('refer-to').getUrl().getCopy()
-            self.ua.equeue.append(CCEventDisconnect(also, rtime = req.rtime, origin = self.ua.origin))
+            also = req.getHFBody('refer-to').getCopy()
+            self.ua.equeue.append(CCEventDisconnect([ also ], rtime = req.rtime, origin = self.ua.origin))
```

Comment:
Is there any reason to allow multiple contacts in the CCEventDisconnect? I see none really.